fixtures/babel/packages/babel-helper-create-class-features-plugin/src/decorators.js

```diff
@@ -0,0 +1,168 @@
+import { types as t, template } from '@babel/core';
```

Comment:
These fixture files from other projects don't need to be reviewed

Reply:
I appreciate you're saying some files don't need to be reviewed, but it's unclear which files you mean?

Reply:
I think he means any file under `fixtures/`, including this one (?)

Reply:
@Josh68 See https://github.com/cloudfour/eslint-config/pull/91#issuecomment-605150154

The _folders_ in `fixtures` can all be ignored. The files directly in fixtures (like the load-repo script) should be reviewed